src/tests.rs

```diff
@@ -4695,22 +4652,9 @@ mod on_idle {
 					initial_stake,
 					final_stake
 				);
-
-				// println!(
-				// 	"Staker {}: Initial stake = {}, Final stake = {}, Increase = {}",
-				// 	staker,
-				// 	initial_stake,
-				// 	final_stake,
-				// 	final_stake - *initial_stake
-				// );
 			}
 
-			let total_processing_blocks = (block_num - processing_start_block) + 1;
-
-			println!(
-				"Successfully verified autocompound processing across {} blocks",
-				total_processing_blocks
-			);
+			let _total_processing_blocks = (block_num - processing_start_block) + 1;
```

Comment:
This parameter is not needed. Either match it against an `assert_eq!` or alike, or better to remove the line.